packages/react-sdk/src/context.tsx

```diff
@@ -49,5 +50,13 @@ export const ReactSdkContext = createContext<
 
 export const useResource = (name: string) => {
   const { resources } = useContext(ReactSdkContext);
-  return resources[name];
+  const resource = useMemo(
+    () =>
+      isPlainObject(resources[name])
+        ? createJsonStringifyProxy(resources[name])
+        : resources[name],
+    [resources, name]
+  );
```

Comment:
Maybe move resources[name] outside of useMemo?

Reply:
Done